### ISSUE TITLE
Support sequence_bias when decoding

### ISSUE THREAD

**Issue by mc-marcocheng** [SYSTRAN/faster-whisper#670]
In huggingface transformers, there is a generate option called `sequence_bias` to increase/decrease the logits of user-specified sequence of tokens, using the `SequenceBiasLogitsProcessor`. Would be nice to have such generation option here as well.
